Detect right-to-left text direction

### User Story
As a developer I want to know when text is going to be displayed right-to-left so that i could adjust views and styles to display that properly.

### Description
For Arabic, Hebrew and other right-to-left languages React Native aligns text to the right. That is breaking layout for chat messages.

As a quickfix we're using regexp to detect most of right-to-left languages, but this method is error-prone and don't cover all the cases https://github.com/status-im/status-react/pull/5054

### Solution
I haven't found the API for Text or View components to tell if text is going to be displayed and aligned as RTL text. Probably there is some API for this on native level (android/ios), needs to be researched further.

I'm ready to test now but there are compilation error that are unrelated to the changes I made (they also occur in the 'develop' branch when compiling).

> Task :app:processDebugManifest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@appComponentFactory value=(android.support.v4.app.CoreComponentFactory) from [com.android.support:support-compat:28.0.0] AndroidManifest.xml:22:18-91
        is also present at [androidx.core:core:1.0.0] AndroidManifest.xml:22:18-86 value=(androidx.core.app.CoreComponentFactory).
        Suggestion: add 'tools:replace="android:appComponentFactory"' to <application> element at AndroidManifest.xml:7:5-117 to override.

I'll keep tabs on the 'develop' branch to see when this is resolved so I can continue with the testing.

--- a/src/status_im/data_store/messages.cljs
+++ b/src/status_im/data_store/messages.cljs
@@ -35,7 +35,7 @@
                        :ens-name (:ensName message)
                        :line-count (:lineCount message)
                        :parsed-text (:parsedText message)
-                       :rtl (:rtl message)
+                       :rtl? (:rtl message)
                        :response-to (:responseTo message)}
              :outgoing (boolean (:outgoingStatus message)))
       (dissoc :ensName :chatId :text :rtl :responseTo :sticker :lineCount :parsedText)))

--- a/src/status_im/data_store/messages_test.cljs
+++ b/src/status_im/data_store/messages_test.cljs
@@ -15,7 +15,7 @@
                                :line-count 1
                                :ens-name "ens-name"
                                :parsed-text "parsed-text"
-                               :rtl false
+                               :rtl? false
                                :response-to "a"}
                      :whisper-timestamp 1
                      :outgoing-status :sending

